### PR TITLE
339 : Adding section

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: training/050_microservices/030_Novice.rst [hunk 1]
```diff
@@ -167,18 +167,76 @@ Changing a Microservice
...
+
+    - `The OAuth Authorization Framework <https://auth0.com/docs/authorization/protocols/protocol-oauth2>`_ 
+
+  - OAuth authorization typically use an access token to grant access to protected ressources owned by a client
+  - setup in a Sesam node uses environment variables defined globally
+  - secrets are defined locally for a specific system
+
+Authentication is the process of verifying that a party is who they claim to be, whilst authorization is the process of verifying the access rights of a party. With regards to microservices the process of authentication can vary quite a bit. Among the most common forms of authentication, a verification via username and password is a typical one. 
+
+After having completed both authentication and authorization, the user will have a set of ressources available. Such ressources can vary depending on what was authenticated against, albeit an API typically has endpoints whereas for example a database will have tables.
```
Comment (gabriellsesam) [inline training/050_microservices/030_Novice.rst:190]: ressources -> resources

File: training/050_microservices/030_Novice.rst [hunk 2]
```diff
@@ -167,18 +167,76 @@ Changing a Microservice
...
+
+API authorization
+##################
+
+APIs use authorization to ensure that a party requests data in a secure way. This will typically involve authenticating that a party of a request have the permission needed to access and/or manipulate the relevant ressource. Authorizing towards an API has historically been done in different ways, albeit recent authorization frameworks ease and standardize this. One of the most popular are the `OAuth Authorization Framework <https://auth0.com/docs/authorization/protocols/protocol-oauth2>`_. This framework is a protocol that allows a party to grant a third-party web site or application access to its protected resources, without necessarily revealing their long-term credentials or even their identity.
```
Comment (gabriellsesam) [inline training/050_microservices/030_Novice.rst:195]: authenticating that a party of a request have the permission needed to access...
->
authenticating that the party which made the request is permitted to access...

File: training/050_microservices/030_Novice.rst (hunk 1 above)
Comment (JonasAls89) [reply training/050_microservices/030_Novice.rst]: Correcting typos

File: training/050_microservices/030_Novice.rst [hunk 3]
```diff
@@ -167,18 +167,76 @@ Changing a Microservice
...
+.. hint::
+
+  - An access token is a string that specifies a scope, lifetime and other access related attributes. 
+  - Access tokens are issued by third-party clients, via an authorization server, approved by a resource owner and distributed to authorized users.
+
+Environment variables and secrets 
+#################################
 
-Passing Env-vars/Secrets i Sesam
+Upon successfully authorizing towards an API, you can now start to request ressources implemented in your microservice code, assuming you have set up your environment variables and secrets. Doing so in a Sesam node is straight forward. Environment variables and secrets can both be defined locally for a given microservice system, albeit it is recommended to define environment variables globally in your Sesam node whereas secrets should be defined locally for your specific microservice system.
```
Comment (gabriellsesam) [inline training/050_microservices/030_Novice.rst]: Why should secrets be defined locally for you speicifc microservice system and not through the secrets overview next to the variables overview?

**Review by gabriellsesam — CHANGES_REQUESTED** [review]
Minor rewriting comment & question

File: training/050_microservices/030_Novice.rst (hunk 2 above)
Comment (JonasAls89) [reply training/050_microservices/030_Novice.rst]: will correct

File: training/050_microservices/030_Novice.rst (hunk 3 above)
Comment (JonasAls89) [reply training/050_microservices/030_Novice.rst]: dunno, it´s recommmended in the docs..

https://docs.sesam.io/concepts.html?highlight=environment%20variables#environment-variables

**Pull request merged**